apps/docs/app/ui/layout/sectionPopover/sectionPopover.css

```diff
@@ -1,25 +1,25 @@
+.hd-section-popover {
+    width: calc(100% - (var(--hd-space-2) * 2));
+    transform: translateX(calc(-1 * calc(var(--hd-space-2) / 2)));
```

Comment:
why do we need this?

Reply:
to center the popover otherwise it stuck to the right

Reply:
containerPadding shouldn't allow it to stick to the right. That width the width should be fine I'd think..

Reply:
I removed the transform.